app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,54 @@
+<resources>
+    <string name="app_name">Simple Music Player</string>
+    <string name="app_launcher_name">Music Player</string>
+    <string name="playlist_empty">Deine Playlist ist leer</string>
+    <string name="no_permissions">Die Berechtigung zum Zugriff auf deine Songs wurde nicht erteilt.</string>
+    <string name="delete">Löschen</string>
+    <string name="edit">Bearbeiten</string>
+    <string name="undo">Rückgängig</string>
+    <string name="title">Titel</string>
+    <string name="artist">Künstler</string>
+    <string name="file_name">Dateiname</string>
+    <string name="extension">Dateiendung</string>
+    <string name="rename_song">Song umbenennen</string>
+    <string name="rename_song_error">Während des Umbenennens ist ein Fehler aufgetreten</string>
+    <string name="rename_song_empty">Fülle bitte alle Felder aus</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Abbrechen</string>
+
+    <plurals name="songs_deleted">
+        <item quantity="one">1 Titel gelöscht</item>
+        <item quantity="other">%1$d Titel gelöscht</item>
+    </plurals>
+
+    <string name="previous" translatable="false">Voriger Titel</string>
```

Comment:
those strings arent visible anywhere, they are used just for giving names to notification controls. Ive made them translatable so its less confusing and maybe have some use in the future, but nobody will see the translations now :)